### PR TITLE
allow quotes and double quotes setting tag value strings

### PR DESCRIPTION
**Pull request by hmueller01**
I got
```
RuntimeError: /bin/sh: -c: line 0: unexpected EOF while looking for matching `''
/bin/sh: -c: line 1: syntax error: unexpected end of file

```
when I used a quote (') in a tag value string. So I have rewritten that a little bit ...

### CODE REVIEW

File: pyexif/__init__.py [hunk 1]
```diff
@@ -277,7 +277,11 @@ def setTags(self, tags_dict):
             raise TypeError('tags_dict is not instance of dict')
         vallist = []
         for tag in tags_dict:
-            vallist.append("-{0}='{1}'".format(tag, tags_dict[tag]))
+            val = tags_dict[tag]
+            # escape double quotes in case of string type
+            if isinstance(val, basestring):
```
Comment (EdLeafe) [inline pyexif/__init__.py]: `basestring` is a Python2-specific class. Instead, use `six.string_types` to support both Python2 and Python3

**Comment by hmueller01** [comment]
You are right. Still using Python2. 😇 Will change this tonight ...

**Comment by hmueller01** [comment]
Ok, I changed it to six.string_types. Tx for the hint.

**Comment by EdLeafe** [comment]
I tested the change locally and it looks fine. One thing I would suggest is to make the same change in the setTag() method as well, since they should both behave the same.

**Comment by hmueller01** [comment]
ok, done.

**Comment by EdLeafe** [comment]
Thanks for the quick work!

**Pull request merged**